File: java-maven/files/src/test/java/{{ .PackagePath }}/{{ .NameWord }}/scenarios/HttpScenario.java
package {{ .Package }}.{{ .NameWord }}.scenarios;

import io.gatling.javaapi.core.ScenarioBuilder;

import static io.gatling.javaapi.core.CoreDsl.scenario;

import {{ .Package }}.{{ .NameWord }}.cases.HttpActions;

public final class HttpScenario {

    private HttpScenario() {
    }

    public static ScenarioBuilder create() {
        return scenario("Http Scenario")
                .exec(HttpActions.getMainPage);
    }
}

File: kotlin-maven/files/src/test/kotlin/{{ .PackagePath }}/{{ .NameWord }}/scenarios/HttpScenario.kt
package {{ .Package }}.{{ .NameWord }}.scenarios

import io.gatling.javaapi.core.CoreDsl.scenario
import io.gatling.javaapi.core.ScenarioBuilder
import {{ .Package }}.{{ .NameWord }}.cases.HttpActions

object HttpScenario {
    fun create(): ScenarioBuilder =
        scenario("Http Scenario")
            .exec(HttpActions.getMainPage)
}
